sf/draft procurement/stmt of qual.md

```diff
@@ -13,15 +13,15 @@ The Smithsonian Institution (SI) intends to release a Request for Proposal on or
 
 The bullet points below ensure that the size of the vendor is adequate for this particular work, and that the anticipated price proposal is in the same 'ballpark' as the government budget.  
 
-* Has your company have adequate size to successfully deliver the digital product (15 FTEs minimum)?
-* Does your company certify that it is able to build the product within the budgetary constraints of the Government. See size estimate of team as indicated in the draft Statements of Objectives?
+* Does your company have adequate size to successfully deliver the digital product (5-7 FTEs minimum)?
```

Comment:
- In the [SOO' known-unknown section](https://github.com/nmaahcdigital/ox/blob/master/sf/draft%20procurement/soo.md) you say "it's hard to imagine a team size of less than 5 or larger than 7, with several of those individuals not being FTEs. We are communicating this to ensure that vendors and SI are in the same ball park for price. This may evolve since SI is still in the market research phase." We're calling this out as potentially conflicting information.